cloud/ec2.go

```diff
@@ -654,7 +654,7 @@ func (m *ec2Manager) ModifyHost(ctx context.Context, h *host.Host, opts host.Hos
 	if opts.NoExpiration != nil {
 		catcher.Add(m.setNoExpiration(ctx, h, *opts.NoExpiration))
 	}
-	if opts.AddHours > 0 {
+	if opts.AddHours != 0 {
```

Comment:
We'll now need protection / testcases to ensure if the expiry length is shortened, it isn't shortened to before the current timestamp

Reply:
The route you mentioned is actually internally calling the same function that checks for 30 days (and it double checks for 14 days so I just removed that check and kept the 30 days check that it was doing).

There were four other instances of MaxSpawnHostExpirationDurationHours. Two that I just removed because of a double check and two in our ModifyVolumeHandler or PATCH /rest/v2/volumes/{volume_id} (https://github.com/ZackarySantana/evergreen/blob/c460a923baacf7002a86a435c72f4b7c88923c5b/rest/route/host_spawn.go#L814). 

I'm not that educated on our volumes and their use cases and what people do with them, or whatnot- should I just change the variable to 30 days to extend volume expiration too? I think it makes sense to align volume expiration with host expiration. The volumes also cannot be reduced in expiration time.

Reply:
let's just keep the scope of this ticket to spawn host extension, we can potentially make a new ticket to investigate if we should do this for volumes too or not.

That being said lets rename `MaxSpawnHostExpirationDurationHours` --> `MaxVolumeExpirationDurationHours` since that's all it's used for now